app/views/patients/update.js.erb

```diff
@@ -1,44 +1,45 @@
-// rerack approx gestation
-<% if @patient.previous_changes.present? %>
-  $('#patient_appointment_date')
-    .parent()
-    .find('span.help-block')
-    .text('Approx gestation at appt: <%= @patient.last_menstrual_period_at_appt %>');
+// rerack fields
+$('#patient_appointment_date')
```

Comment:
The callback wipes `previous_changes`, so we had to rewire this a little (a lot). Now it just rerenders everything after save regardless.